Python/Product/PythonTools/visualstudio_py_debugger.py

```diff
@@ -2318,7 +2318,7 @@ def detach_threads():
 
 def new_thread(tid = None, set_break = False, frame = None):
     # called during attach w/ a thread ID provided.
-    if tid == debugger_thread_id:
+    if tid and tid == debugger_thread_id:
```

Comment:
When new_thread() is called, tid is None and then the comparison tid==debugger_thread_id evaluates to true and new_thread returns with a None. I assume that this is not intended.